backend/src/graphql/arg/SearchUsersArgs.ts

```diff
@@ -11,9 +12,6 @@ export default class SearchUsersArgs {
   @Field(() => Int, { nullable: true })
```

Comment:
```suggestion
  @Field(() => Int, { default: 25 })
```
This should never be null

Reply:
I tried this in the follow up (sub) PR, but got an error which I couldn't solve easy.